assets/app/components/site/editor/editorContainer.js

```diff
@@ -22,7 +22,7 @@ const propTypes = {
 };
 
 const redirectToFileOnBranch = (siteId, branch, filePath) => {
-  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`);
+  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`, 'replace');
```

Comment:
We can have two different methods, but I believe the concepts of `push` and `replace` will still need to be communicated. The consumer of these methods will want to know whether or not their redirect is adding an entry to the browser's history or replacing the existing entry with a new one.